coral.rst

```diff
@@ -1,10 +1,10 @@
 .. _coral2:
```

Comment:
does this also need to be changed to just `coral`? I don't think it renders on the .rst page, so probably not a big deal if you don't change it. 🙂

Reply:
Thanks @cmoussa1 for catching that.  Just pushed a fix.